content/advent-2015/reducing-boilerplate-with-go-generate.md

```diff
@@ -0,0 +1,294 @@
++++
+author = ["Ernesto Jiménez"]
+date = "2015-12-08T08:00:00+00:00"
+title = "Reducing boilerplate with go generate"
+series = ["Advent 2015"]
++++
+
+Go is an awesome language. It's simple, powerful, has great tooling and
+many of really enjoy using it every day. However, as it's common with
+strongly typed languages, we write a good deal of boilerplate to
+connect things around.
+
+In this post we'll have cover mostly three points:
+
+1. Why can we build tools with Go that will help reduce boilerplate using code generation.
```

Comment:
Why we can build tools with Go that will help reduce boilerplate using code generation.